### PR TITLE
#64 Travis-ci configuration to enable pre-build checks

### PR DESCRIPTION
**Pull request by longtimeago**
#64 Travis-ci configuration to enable pre-build checks

### CODE REVIEW

File: download-maven-plugin/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+sudo: false
+cache:
+  directories:
+    - $HOME/.m2
+env:
+  global:
+    - MAVEN_OPTS="-XX:MaxPermSize=256m -Xmx512m"
+    - JAVA_OPTS="-XX:MaxPermSize=256m -Xmx512m"
+jdk:
+  - oraclejdk7
```
Comment (mickaelistria) [inline download-maven-plugin/.travis.yml]: Why 2 jdks?

Comment (longtimeago) [inline download-maven-plugin/.travis.yml]: @mickaelistria I know many issues based on jdk type. Prebuild for both will show us those issues rapidly.

Comment (mickaelistria) [inline download-maven-plugin/.travis.yml]: Ok, why not ;)

Comment (longtimeago) [inline download-maven-plugin/.travis.yml]: @mickaelistria we could add more versions here, for example for java8, or for 6 if we support them

Comment (mickaelistria) [inline download-maven-plugin/.travis.yml]: Let's start with those 2 ones, and see if it's worth adding some more later.

**Comment by mickaelistria** [comment]
Rebased and merged as e08463a775e3783f79cebb973a7bf72f340b79e8 . Thank you!

**Pull request closed without merging**